extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -39,13 +100,74 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
   layoutCallback() {
     return Services.storyStoreServiceForOrNull(this.win).then(
       (storeService) => {
-        this.element.addEventListener('click', () =>
-          storeService.dispatch(Action.TOGGLE_MUTED, false)
-        );
+        this.storeService_ = storeService;
+        this.initializeListeners_();
       }
     );
   }
 
+  /** @private */
+  maybeInitializeDefaultSticker_() {
+    const defaultSticker = this.getDefaultSticker_();
+    if (!defaultSticker) {
+      return;
+    }
+
+    // Remove all existing child elements and add only pretap and posttap elements.
+    this.element.replaceChildren();
```

Comment:
This might not be necessary (if there are no children) but not too bad to keep it.

Reply:
Yeah this is just in case developers might put random stuff inside the element.